ios/Classes/FlutterWebviewPlugin.m

```diff
@@ -32,27 +49,97 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
     }
 }
 
-- (void)showWebView:(FlutterMethodCall*)call {
+- (void)launch:(FlutterMethodCall*)call {
     NSString *url = call.arguments[@"url"];
-    NSNumber *withJavascript = call.arguments[@"withJavascript"];
+    
+    NSURLRequest *request = [NSURLRequest requestWithURL:[NSURL URLWithString:url]];
+    [self.webview loadRequest:request];
+}
+
+- (void)initWebView:(FlutterMethodCall*)call {
+    // NSNumber *withJavascript = call.arguments[@"withJavascript"];
```

Comment:
I have no idea, but can we deactivate/activate javascript for iOS webview ?

Reply:
I don't think it's easy.